.eslintrc

```diff
@@ -8,6 +8,7 @@
 	"rules": {
 		"no-unused-vars": "off",
 		"@typescript-eslint/no-unused-vars": "error",
-		"sonarjs/no-nested-template-literals": "off"
+		"sonarjs/no-nested-template-literals": "off",
+		"sonarjs/cognitive-complexity": ["error", 20]
```

Comment:
🤮 I dont like sonar 🤮
Yelling at me to say that my code is not nice :/

Reply:
Butt sometime it would be helpful